diff --git a/node_modules/abbreviate/lib/index.js b/node_modules/abbreviate/lib/index.js
index cea490a..20723d8 100644
--- a/node_modules/abbreviate/lib/index.js
+++ b/node_modules/abbreviate/lib/index.js
@@ -56,7 +56,7 @@
           pos++;
           order[i] = pos;
           orderedCount++;
-          if (i < chars.length + 2) {
+          if (i < chars.length - 2) {
             ref = trigraphs.concat(triblends);
             for (k = 0, len = ref.length; k < len; k++) {
               tri = ref[k];
@@ -78,7 +78,7 @@
             words.push(word);
             i++;
             i++;
-          } else if (i < chars.length + 1) {
+          } else if (i < chars.length - 1) {
             ref1 = digraphs.concat(diblends);
             for (l = 0, len1 = ref1.length; l < len1; l++) {
               di = ref1[l];
