diff --git a/node_modules/core-js/internals/global.js b/node_modules/core-js/internals/global.js
index 31fe0db..a154678 100644
--- a/node_modules/core-js/internals/global.js
+++ b/node_modules/core-js/internals/global.js
@@ -11,5 +11,6 @@ module.exports =
   // eslint-disable-next-line no-restricted-globals -- safe
   check(typeof self == 'object' && self) ||
   check(typeof global == 'object' && global) ||
+  check(typeof this == 'object' && this) ||
   // eslint-disable-next-line no-new-func -- fallback
-  (function () { return this; })() || this || Function('return this')();
+  (function () { return this; })() || Function('return this')();
